packages/web/docs/src/components/stud.tsx

```diff
@@ -1,13 +0,0 @@
-import { cn } from '@theguild/components';
-
-export function Stud(props: React.HTMLAttributes<HTMLElement>) {
```

Comment:
Removed it because we already have it in Components, and it had just one use case left.